run.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"gopkg.in/errgo.v1"
 
-	"github.com/Scalingo/go-scalingo/v6/http"
+	httppkg "github.com/Scalingo/go-scalingo/v6/http"
```

Comment:
Why the package renaming? I don't see any conflict in package imports
```suggestion
	"github.com/Scalingo/go-scalingo/v6/http"
```

Reply:
I tried something and didn't revert this renaming change :disappointed: